src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -341,7 +347,7 @@ private string GetRuntimeFrameworkVersion(ITaskItem frameworkReference, KnownFra
                 }
                 if (useLatestRuntimeFrameworkVersion == null)
                 {
-                    useLatestRuntimeFrameworkVersion = TargetLatestRuntimePatch;
+                    useLatestRuntimeFrameworkVersion = TargetLatestRuntimePatch || TargetLatestRuntimePatchIsDefault;
```

Comment:
This impacts both packages to download (what I was after) and runtime config version (which is a bug I introduced, now we're targeting latest in FDD.)

Reply:
Fixed with do-over in 2cc2b8a